sqldelight-idea-plugin/src/main/kotlin/com/squareup/sqldelight/intellij/SqlDelightRenameProcessor.kt

```diff
@@ -12,7 +12,7 @@ import com.squareup.sqldelight.core.SqlDelightFileIndex
 import com.squareup.sqldelight.core.lang.SqlDelightFile
 import com.squareup.sqldelight.core.lang.psi.StmtIdentifier
 import com.squareup.sqldelight.core.lang.psi.StmtIdentifierMixin
-import org.jetbrains.kotlin.idea.refactoring.toPsiFile
+import org.jetbrains.kotlin.idea.core.util.toPsiFile
```

Comment:
im assuming this function will be on the kotlin intellij plugin version 1.3.50 classpath so the package change should be okay